packages/vscoq-language-server/vscoq-language-server.2.1.7/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" "Maxime Dénès" "Romain Tetley" ]
+license: "MIT"
+homepage: "https://github.com/coq-community/vscoq"
+bug-reports: "https://github.com/coq-community/vscoq/issues"
+dev-repo: "git+https://github.com/coq-community/vscoq"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" { >= "4.14" }
+  "dune" { >= "3.5" }
+  "coq-core" { ((>= "8.18" < "8.21") | (= "dev")) }
+  "coq-stdlib" { ((>= "8.18" < "8.21") | (= "dev")) }
+  "yojson"
+  "jsonrpc" { >= "1.15"}
+  "ocamlfind"
+  "ppx_inline_test"
+  "ppx_assert"
+  "ppx_sexp_conv"
+  "ppx_deriving"
+  "sexplib"
+  "ppx_yojson_conv"
+  "ppx_import"
+  "ppx_optcomp"
+  "result" { >= "1.5" }
+  "lsp" { >= "1.15" < "1.19"}
+  "sel" {>= "0.4.0"}
+]
+synopsis: "VSCoq language server"
```

Comment:
```suggestion
synopsis: "VSCoq language server"
available: arch != "arm32" & arch != "x86_32"
```
Due to
```
#=== ERROR while compiling vscoq-language-server.2.1.7 ========================#
# context              2.3.0~alpha~dev | linux/x86_32 | ocaml-base-compiler.5.2.0 | pinned(https://github.com/coq-community/vscoq/releases/download/v2.1.7/vscoq-language-server-2.1.7.tar.gz)
# path                 ~/.opam/5.2/.opam-switch/build/vscoq-language-server.2.1.7
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p vscoq-language-server -j 31
# exit-code            1
# env-file             ~/.opam/log/vscoq-language-server-7-17ee2d.env
# output-file          ~/.opam/log/vscoq-language-server-7-17ee2d.out
### output ###
# (cd _build/default && /home/opam/.opam/5.2/bin/ocamlc -linkall -g -bin-annot -bin-annot-occurrences -I vscoqtop/.vscoqtop.eobjs/byte -I /home/opam/.opam/5.2/lib/base -I /home/opam/.opam/5.2/lib/base/base_internalhash_types -I /home/opam/.opam/5.2/lib/base/caml -I /home/opam/.opam/5.2/lib/base/shadow_stdlib -I /home/opam/.opam/5.2/lib/coq-core/boot -I /home/opam/.opam/5.2/lib/coq-core/clib -I /home/opam/.opam/5.2/lib/coq-core/config -I /home/opam/.opam/5.2/lib/coq-core/engine -I /home/opam/.opam/5.2/lib/coq-core/gramlib -I /home/opam/.opam/5.2/lib/coq-core/interp -I /home/opam/.opam/5.2/lib/coq-core/kernel -I /home/opam/.opam/5.2/lib/coq-core/lib -I /home/opam/.opam/5.2/lib/coq-core/library -I /home/opam/.opam/5.2/lib/coq-core/parsing -I /home/opam/.opam/5.2/lib/coq-core/perf -I /home/opam/.opam/5.2/lib/coq-core/pretyping -I /home/opam/.opam/5.2/lib/coq-core/printing -I /home/opam/.opam/5.2/lib/coq-core/proofs -I /home/opam/.opam/5.2/lib/coq-core/sysinit -I /home/opam/.opam/5.2/lib/coq-core/tactics -I /home/opam/.opam/5.2/lib/coq-core/vernac -I /home/opam/.opam/5.2/lib/coq-core/vm -I /home/opam/.opam/5.2/lib/findlib -I /home/opam/.opam/5.2/lib/jsonrpc -I /home/opam/.opam/5.2/lib/lsp -I /home/opam/.opam/5.2/lib/ocaml/dynlink -I /home/opam/.opam/5.2/lib/ocaml/str -I /home/opam/.opam/5.2/lib/ocaml/threads -I /home/opam/.opam/5.2/lib/ocaml/unix -I /home/opam/.opam/5.2/lib/parsexp -I /home/opam/.opam/5.2/lib/ppx_deriving/runtime -I /home/opam/.opam/5.2/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/5.2/lib/ppx_yojson_conv_lib -I /home/opam/.opam/5.2/lib/sel -I /home/opam/.opam/5.2/lib/seq -I /home/opam/.opam/5.2/lib/sexplib -I /home/opam/.opam/5.2/lib/sexplib0 -I /home/opam/.opam/5.2/lib/uutf -I /home/opam/.opam/5.2/lib/yojson -I /home/opam/.opam/5.2/lib/zarith -I dm/.dm.objs/byte -I language/.language.objs/byte -I protocol/.protocol.objs/byte -intf-suffix .ml -no-alias-deps -open Dune__exe -o vscoqtop/.vscoqtop.eobjs/byte/dune__exe__LspManager.cmo -c -impl vscoqtop/lspManager.ml)
# File "vscoqtop/lspManager.ml", line 39, characters 28-38:
# 39 | let max_memory_usage  = ref 4000000000
#                                  ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type "int"
```

Reply:
Yes ! Thank you ! I can fix it in the next release